src/tests/ftest/pytorch/dataset.yaml

```
@@ -0,0 +1,44 @@
+hosts:
+  test_servers: 1
+  test_clients: 1
+server_config:
```

We need a timeout here. The last run took ~90s so I'd recommend 180 for cushion. I'll see about adding a lint check for this so it's caught early
```suggestion
  test_clients: 1
timeout: 180
server_config:
```

---

I'll fix this specifically in a follow-on if there aren't any other changes coming. Would be great to get this landed as-is so that we don't need to rebase other work.

---

@enakta pls push this in another PR.

---

@johannlombardi https://github.com/daos-stack/daos/pull/15829 created